aviary/subsystems/propulsion/propeller/hamilton_standard.py

```diff
@@ -595,8 +595,7 @@ def setup(self):
         )  # blade integrated lift coeff
 
         self.add_output('thrust_coefficient', val=np.zeros(nn), units='unitless')
-        self.add_output('blade_angle', val=np.zeros(nn), units='deg')
```

Comment:
You've removed `blade_angle` as an output, but in hamilton_standard.png it still shows blade_angle as an output.

Reply:
I removed the `blade_angle` from `hamilton_standard.png`.